packages/market/src/blockchain-facade/Demand.ts

```diff
@@ -51,6 +51,7 @@ export interface IDemand extends IDemandOnChainProperties {
     id: string;
     offChainProperties: IDemandOffChainProperties;
     fill: (entityId: string) => Promise<TransactionReceipt>;
+    fillAt: (entityId: string, energy: number) => Promise<TransactionReceipt>;
```

Comment:
Can't `fill()` and `fillAt` be the same function `fillAt: (entityId: string, energy?: number)`?